Dockerfile

```diff
@@ -44,6 +44,21 @@ RUN md5sum -b oc-hc-v0.1.3-linux-amd64.tar.gz | grep $OC_HC_MD5
 # Extract the binary
 RUN tar xzf oc-hc-v0.1.3-linux-amd64.tar.gz --directory /out
 
+
+### Attach yq binary into the image
+ENV YQ_URL="https://github.com/mikefarah/yq/releases/download/v4.35.1/yq_linux_arm64.tar.gz"
+RUN mkdir -p /yq
+WORKDIR /yq
+
+# Download the yq release
+RUN curl -sSLf -O $YQ_URL
+
+# Check md5sum for the downloaded tar
+RUN md5sum yq_linux_arm64.tar.gz  | md5sum -c
```

Comment:
What do we want for this md5sum?

Reply:
Added the static MD5 value and `grep` it